### PR TITLE
[Merged by Bors] - chore: move `NNRat` APIs earlier

### PR DESCRIPTION
**Pull request by astrainfinita**
No need to import bundled ordered algebraic typeclasses in `ring` tactic now.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/NNRat/Defs.lean [hunk 1]
```diff
@@ -14,7 +14,7 @@ import Mathlib.Algebra.Ring.Rat
 This file defines the nonnegative rationals as a subtype of `Rat` and provides its basic algebraic
 order structure.
 
-Note that `NNRat` is not declared as a `Field` here. See `Data.NNRat.Lemmas` for that instance.
+Note that `NNRat` is not declared as a `Semifield` here. See `Algebra.Field.Rat` for that instance.
```
Comment (Ruben-VandeVelde) [inline Mathlib/Data/NNRat/Defs.lean:17]:
```suggestion
Note that `NNRat` is not declared as a `Semifield` here. See `Mathlib.Algebra.Field.Rat` for that instance.
```

**Review by Ruben-VandeVelde — APPROVED** [review]
Seems sensible.

maintainer merge

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Comment by Vierkantor** [comment]
Please fix the build failure, and feel free to merge then.

bors d+

**Pull request closed without merging**